packages/core/src/components/radio-pill/_radio-pill.scss

```diff
@@ -0,0 +1,67 @@
+@import '../../global/variables';
+
+@include exports('ray-radio-pill') {
+  .#{$ray-class-prefix}radio-pill__wrapper {
+    width: 100%;
+    display: flex;
+  }
+
+  .#{$ray-class-prefix}radio-pill {
+    float: left;
+    display: flex;
+    flex-grow: 1;
+    text-align: center;
+    align-items: center;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
```

Comment:
these unfortunately won't work without a width attribute on the flexed element

Reply:
fixed

Reply:
![Screen Shot 2019-05-08 at 12 52 26 PM](https://user-images.githubusercontent.com/17712022/57393000-2c103600-7190-11e9-9c4e-242991d63ea7.png)